bootstrap/config/config_test.go

```diff
@@ -378,21 +378,51 @@ func TestFindChangedKey(t *testing.T) {
 }
 
 func TestGetConfigFileLocation(t *testing.T) {
-	dir := "myRes"
-	profile := "myProfile"
-	file := "myFile.yaml"
-	expected := filepath.Join(dir, profile, file)
-	defer os.Clearenv()
+	tests := []struct {
+		name       string
+		dir        string
+		profile    string
+		path       string
+		secretName string
+		expected   string
+	}{
+		{
+			name:     "valid - file",
+			dir:      "myRes",
+			profile:  "myProfile",
+			path:     "myFile.yaml",
+			expected: filepath.Join("myRes", "myProfile", "myFile.yaml"),
```

Comment:
Please add other variations of file paths like absolute `/res/myFile.yaml` and relative `./res/myFile.yaml` and ../../res/myFile.yaml`